.github/workflows/test_dev_env.yml

```diff
@@ -18,4 +19,85 @@ jobs:
           docker-compose up -d
           sleep 30
           docker ps -a | (! grep Exited ) # Return a non-zero exit code if any of the containers are stopped
+  build:
+    name: Verify build
+    runs-on: ubuntu-latest
+    env:
+      POSTGRES_USER: daria
+      POSTGRES_PASSWORD: ""
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_USER: daria
+          POSTGRES_DB: dcaf_case_management_test
+          POSTGRES_HOST_AUTH_METHOD: trust
+        ports:
+          - 5432:5432
+        # Add a health check
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      # Ruby setup
+      - name: Setup ruby
+        uses: ruby/setup-ruby@v1 # automagically reads from .ruby-version
+        with:
+          bundler-cache: true
+
+      # npm/yarn setup
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.2
+      - name: Find yarn cache location
```

Comment:
For caching go ahead and try this:

```
- uses: actions/cache@v2
  with:
    path: '**/node_modules'
    key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
```

Reply:
https://github.com/DCAFEngineering/dcaf_case_management/pull/2361/commits/5080a9e0d02c43a9508e42050f216a7e42de439a addresses this, I think? I popped out the other two cache commands and am using your snippet here instead.